src/main/java/tech/aroma/data/sql/SQLStatements.java

```diff
@@ -52,6 +52,8 @@
         static final String REACTIONS = loadSQLFile("tech/aroma/sql/deletes/delete_reactions.sql");
         static final String TOKEN = loadSQLFile("tech/aroma/sql/deletes/delete_token.sql");
         static final String USER = loadSQLFile("tech/aroma/sql/deletes/delete_user.sql");
+        static final String USER_DEVICE = loadSQLFile("tech/aroma/sql/deletes/delete_user_device.sql");
+        static final String USER_DEVICES = loadSQLFile("tech/aroma/sql/deletes/delete_all_user_devices.sql");
```

Comment:
Rename to `ALL_USER_DEVICES`; just makes more sense.
![bitmoji](https://render.bitstrips.com/v2/cpanel/10211502-288439948_4-s1-v1.png?transparent=1&palette=1&width=246)